src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -129,4 +133,10 @@ public RobotType getRobotType() {
 	public boolean isCompetition() {
 		return getRobotType() == RobotType.COMPETITION;
 	}
+
+	public void registerCommands() {
+		// parameters: Str command name, Command command
+
+		// NamedCommands.registerCommand(null, null);
```

Comment:
Please add a comment noting that these are commands intended for use in Autonomous, so not every single command gets listed here.